ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -32,21 +32,27 @@
         workspaceActions: '.choice-actions'
     },
     onRender: function() {
-        if (this.model.get('type') === 'basic' || this.model.get('type') === 'text') {
+        if (
+            this.model.get('type') === 'basic' ||
+            this.model.get('type') === 'text'
```

Comment:
✏️ I feel like this refactor is a bit odd

Reply:
I need a new formatter..